test/RibbonOptionsVault.js

```diff
@@ -1250,12 +1328,15 @@ function behavesLikeRibbonOptionsVault(params) {
           params.asset,
           this.oracle,
           await this.vault.currentOptionExpiry(),
-          BigNumber.from("148000000000").sub(BigNumber.from("1"))
+          this.isPut
+            ? BigNumber.from("148000000000000000000")
+            : BigNumber.from("148000000000").sub(BigNumber.from("1"))
```

Comment:
Hmm just checking on this, are we sure that we have more decimal places for the oracle price in puts?

Reply:
I changed it to a misleading number with the zeros  -- just needed the PUT to expire OTM so higher than strike price. since before you were testing selling calls you wanted that number to be smaller (not exactly sure what 148000000000 is -- 1480?) so I just changed it to be bit bigger just now and it works (extra 0)

Reply:
Oh yea. This is hardcoded to 1480 * 10**18, which was a ITM expiry price for ETH. I suspect why it's ITM is because it's Bitcoin. To make this clearer we can make the ITM settlement price as part of the `params`

Reply:
only seeing 9 0's in there so makes sense btc is itm but eth also passes this